src/app/api/fetchData.js

```diff
@@ -0,0 +1,13 @@
+import { countriesAxios } from "@/countriesAxios"
+
+const fetchData = async (endpoint, options = {filters: ''}) => {
```

Comment:
```suggestion
const fetchData = async (endpoint, options: {filters}) => {
```

Dzięki powyższemu możemy użyć `filters` od razu. Domyślna wartość nie będzie też konieczna jeżeli przekażesz zawsze pustą tablicę, jeżeli nie podasz `filters` w opcjach w hooku, o czym piszę w komentarzu niżej.

Reply:
Zrobiłem  `options = {filters}` ponieważ zapis `options: {filters}` powodował błedy

Reply:
Cofam moj komentarz, dalem samo `options` i juz wszystko działa. Wiem czemu, ponieważ zapis `options: {filters}` jest pod TS a ja robie w JS. Troche zamieszaleme sie